docs/user/user/pings/sent-by-glean.md

```diff
@@ -0,0 +1,59 @@
+# Pings sent by Glean
+
+If data collection is enabled, the Glean SDK provides a set of built-in pings that are assembled out of the box without any developer intervention.  The following is a list of these built-in pings:
+
+- [`baseline` ping](baseline.md): A small ping sent every time the application goes to foreground and background. Going to foreground also includes when the application starts.
+- [`deletion-request` ping](deletion_request.md): Sent when the user disables telemetry in order to request a deletion of their data.
+- [`events` ping](events.md): The default ping for events. Sent every time the application goes to background or a certain number of events is reached.
+- [`metrics` ping](metrics.md): The default ping for metrics. Sent approximately daily.
+
+Applications can also define and send their own [custom pings](custom.md) when the schedules of these pings is not suitable.
+
+There is also a [high-level overview](ping-schedules-and-timings.html) of how the `metrics` and `baseline` pings relate and the timings they record.
+
+## Defining foreground and background state
```

Comment:
Should this section be part of the `baseline` ping page?

Reply:
I'm not sure, it really applies to at least both the baseline and the events ping. Should I copy it to both pages?

Reply:
Hum, you are right... Let's leave it here, but do make sure that references to these terms are linkfied I guess.